ingest/downloader/downloader.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+
+from openpyxl import Workbook
+from openpyxl.worksheet.worksheet import Worksheet
+
+from ingest.downloader.flattener import Flattener
+from ingest.importer.spreadsheet.ingest_worksheet import START_DATA_ROW
+
+HEADER_ROW_NO = 4
+
+
+class XlsDownloader:
+    def __init__(self):
+        self.row = HEADER_ROW_NO
+        self.flattener = Flattener()
+
+    def convert_json(self, metadata_list: List[dict]):
+        return self.flattener.flatten(metadata_list)
+
+    def create_workbook(self, input_json: dict) -> Workbook:
+        workbook = Workbook()
+        workbook.remove(workbook.active)
+
+        for ws_title, ws_elements in input_json.items():
+            if ws_title == 'Project':
+                worksheet: Worksheet = workbook.create_sheet(title=ws_title, index=0)
+            else:
+                worksheet: Worksheet = workbook.create_sheet(title=ws_title)
+
+            self.add_worksheet_content(worksheet, ws_elements)
+
+        return workbook
+
+    def add_worksheet_content(self, worksheet, ws_elements: dict):
+        headers = ws_elements.get('headers')
+        self.__add_header_row(worksheet, headers)
+        all_values = ws_elements.get('values')
+
+        self.row = START_DATA_ROW - 1
+        for row_values in all_values:
```

Comment:
Same here. Use enumerate

Reply:
`self.row` is not starting from `0` or `1`. It would be a bit weird code to use enumeration here, I guess.

Reply:
use the `start` argument of `enumerate()`